packages/react-devtools-shared/src/__tests__/preprocessor.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const pathToTransformReactVersionPragma = require.resolve(
+  '../../../../scripts/babel/transform-react-version-pragma',
+);
+
+module.exports = {
+  devtoolsPlugins: [pathToTransformReactVersionPragma],
+};
```

Comment:
nit: I wonder if this file would be better somewhere like `scripts/jest/devtools/preprocessor` ? Closer to the other Jest config stuff

Reply:
Put it here to be close to the setup env but I'll move it!